tests/test_channels/test_partial_transpose.py

```diff
@@ -708,26 +708,12 @@ def test_partial_transpose_cvxpy():
     np.testing.assert_equal(isinstance(x_pt, Vstack), True)
 
 
-def test_partial_transpose_non_square():
```

Comment:
Since we are going back to using the original partial transpose function (with the indices adapted as you've done of course), we may want to include the "non-square" test (with the `sys` argument changed) like so:

```python
def test_partial_transpose_non_square():
    """Test partial transpose on non square matrices ."""
    rho = np.kron(np.eye(2, 3), np.ones((2, 3)))
    rho = np.kron(rho, np.eye(2, 3))

    dim = np.array([[2, 2, 2], [3, 3, 3]])

    res = partial_transpose(rho, sys=1, dim=dim)

    expected = np.kron(np.eye(2, 3), np.ones((3, 2)))
    expected = np.kron(expected, np.eye(2, 3))
    np.testing.assert_equal(np.allclose(res, expected), True)
```